rancher2/structure_cluster_rke_config_cloud_provider_openstack_test.go

```diff
@@ -358,10 +331,7 @@ func TestExpandClusterRKEConfigCloudProviderOpenstackMetadata(t *testing.T) {
 		if err != nil {
 			t.Fatalf("[ERROR] on expander: %#v", err)
```

Comment:
Since we using the testify/assert pkg let's also update these calls to `tf.Fatalf` to an assert call on test failure. Something like https://pkg.go.dev/github.com/stretchr/testify/assert#Assertions.FailNow with the same error msgs. Re-test to make sure behavior hasn't changed

Reply:
Done